app/helpers/application_helper.rb

```diff
@@ -1,2 +1,5 @@
 module ApplicationHelper
+  def model_attributes(klass)
+    @model_attributes ||= klass.column_names.reject{|col| col.include?("_id") || col.include?("created_at") || col.include?("updated_at")}
```

Comment:
lets put this over  a few lines since it seems a bit long
```suggestion
    @model_attributes ||= klass.column_names.reject do |col|
      col.include?("_id") || col.include?("created_at") || col.include?("updated_at")
    end
```

Reply:
i think we should also write a test for this.